src/Tizen.NUI.Components/Controls/Notification.cs

```diff
@@ -283,6 +274,43 @@ public void ForceQuit()
             ClearAll();
         }
 
+        /// <inheritdoc/>
+        /// <since_tizen> 8 </since_tizen>
+        protected override void Dispose(DisposeTypes type)
+        {
+            if (disposed)
+            {
+                return;
+            }
+
+            if (notificationWindow != null)
+            {
+                notificationWindow.Dispose();
+            }
+
+            if (timer != null)
+            {
+                timer.Dispose();
+            }
+
+            if (positionSize != null)
+            {
+                positionSize.Dispose();
+            }
+
+            if (onPostAnimation != null)
+            {
+                onPostAnimation.Dispose();    
+            }
+
+            if (onDismissAnimation != null)
+            {
+                onDismissAnimation.Dispose();    
+            }
+
+            base.Dispose(type);
+        }
+
```

Comment:
When calling only one method immediately after null check, it is better to use null conditional operator.

```C#
notificationWindow?.Dispose();
timer?.Dispose();
positionSize?.Dispose();
onPostAnimation?.Dispose();
onDismissAnimation?.Dispose();
```

This improves code readability, reduces code, and actually reduces instructions in IL.

Reply:
I tried to use "?" here, but when I built it the Code Analysis created [CA2213](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca2213?view=vs-2019) warnings on it. (I don't know why...)

I wanted to keep no Code Analysis warnings in the Notification class so I had to use `if` for null checking.

Reply:
I think my code anaylzer is an old one.

Reply:
Done fixing!